iohub/display_utils.py

```diff
@@ -0,0 +1,149 @@
+# Utility functions for displaying data
```

Comment:
Module docstrings should be string literals. See [PEP 257](https://peps.python.org/pep-0257/).